examples/tina-self-hosted-demo/tina/config.tsx

```diff
@@ -53,6 +47,30 @@ const config = defineStaticConfig({
   },
   schema: {
     collections: [
+      {
+        applicationData: true,
```

Comment:
What is this flag used for?

Reply:
That controls whether the collection writes to git or not. Haven't come up with a better name for it yet

Reply:
is it read only then?

Reply:
No.. it writes to the DB only